### DISCUSSION TITLE
It seems like some HTML video flag attributes are not supported

### DISCUSSION THREAD

**Discussion by clytras** [alpinejs/alpine#3053] [7. Bugs (archived)] [answered]
I can't get `muted` HTML5 video tag to work with something like:

```js
<video
  :muted="muted"
  playsinline
  autoplay
  loop
  x-data="{ muted: true }"
  @click="muted = !muted"
>
</video>
```

As a workaround, I'm shifting the flag using a ref like:

```js
<video
  muted
  playsinline
  autoplay
  loop
  x-ref="video"
  @click="$refs.video.muted = !$refs.video.muted"
>
</video>
```

**Comment by SimoTod** [comment, marked as answer]
I don't think that changing the value of an attribute on the video tag does anything at runtime, it's not Alpine, it's the way that tag works. The attribute is just read when the page loads to set the property with the same name. 
After that point, you need to change the property like your second example.
Note, you don't need a self referencing $refs, in Alpine v3 $el is always the current element

**Reply by clytras** [reply]
Awesome stuff, thanks!
